spyder/workers/updates.py

```diff
@@ -94,7 +94,8 @@ def start(self):
 
         try:
             if hasattr(ssl, '_create_unverified_context'):
-                # Fix for issue # 2685 [Works only with Python >=2.7.9]
+                # Fix for # See spyder-ide/spyder#90442685
```

Comment:
@oscargus is this a typo?

Was it like this?

`See spyder-ide/spyder#2685`

Reply:
Not sure what happened there...